src/Examples/UnitModels/Reactors/cstr_testing.ipynb

```diff
@@ -83,9 +83,8 @@
     "from pyomo.network import Arc\n",
     "\n",
     "from idaes.core import FlowsheetBlock\n",
-    "from idaes.models.properties.modular_properties.base.generic_property import (\n",
-    "        GenericParameterBlock)\n",
-    "from idaes.models.properties.modular_properties.base.generic_reaction import (\n",
+    "from idaes.models.properties.modular_properties import (\n",
```

Comment:
It looks like updating (in somes cases, reverting) these imports to `from idaes.models.properties.modular_properties.base.generic_property import GenericParameterBlock` will resolve the visible issues in the checks.

Reply:
See https://github.com/IDAES/idaes-pse/pull/942

Reply:
Ah, I see. That should resolve it then, thanks for the link.